lib/fizzy/execute.cpp

```diff
@@ -176,6 +176,16 @@ Instance instantiate(const Module& module)
     // NOTE: fill it with zeroes
     bytes memory(memory_min * page_size, 0);
 
+    // Fill out memory based on data segments
+    for (auto const& data : module.datasec)
+    {
+        // FIXME: assert in parser
+        assert(data.memidx == 0);
+        // NOTE: these instructions can overlap
+        assert((data.offset + data.init.size()) <= (memory_max * page_size));
+        __builtin_memcpy(memory.data() + data.offset, data.init.data(), data.init.size());
```

Comment:
I guess could just use `std::memcpy` here?

Reply:
I think `memcpy` or `std::copy` would be better